### PR TITLE
Recycling mese tools by melting

### PR DESCRIPTION
**Pull request by The4codeblocks**
Logic used:
Mese blocks conduct (mese is inherently conductive)
Mesecon only uses mese in its `type = "cooking"` recipe (mesecon is just made of mese)
<mat\> → <mat\> `type = "cooking"` recipes impliy either melting or some other thermal deformation like for iron tool recycling
*This only works when `mesecons_wires` is enabled as if it provides meltable mese logic*

### CODE REVIEW

File: crafts.lua [hunk 1]
```diff
@@ -129,6 +129,12 @@ local tool_materials = {
 	["mithril"] = {"moreores:mithril_ingot", 1, "moreores"},
 }
 
+-- assume mese unmeltable unless mesecons_wires exists
+if core.get_modpath("mesecons_wires") then
```
Comment (OgelGames) [inline crafts.lua]: `crafts.lua:133:4: accessing undefined variable core`
```suggestion
if minetest.get_modpath("mesecons_wires") then
```

**Comment by OgelGames** [comment]
Makes sense to me :+1:

File: crafts.lua (hunk 1 above)
Comment (The4codeblocks) [reply crafts.lua]: I guess changing the whole namespace thing belongs in another pr

**Comment by BuckarooBanzay** [comment]
lgtm :+1:

**Pull request merged**